jerry-core/vm/vm.c

```diff
@@ -2539,6 +2539,17 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
                   || JERRY_CONTEXT (debugger_stop_context) == JERRY_CONTEXT (vm_top_context_p)))
           {
             jerry_debugger_breakpoint_hit (JERRY_DEBUGGER_BREAKPOINT_HIT);
+            if (JERRY_CONTEXT (debugger_flags) & JERRY_DEBUGGER_THROW_ERROR_FLAG)
```

Comment:
I think there are multiple `jerry_debugger_breakpoint_hit`s in the vm. What happns when the others are called and an exception is thrown?